moduleroot/.rubocop.yml.erb

```diff
@@ -0,0 +1,10 @@
+---
+# Managed by modulesync - DO NOT EDIT
+# https://voxpupuli.org/docs/updating-files-managed-with-modulesync/
+
+inherit_gem:
+  voxpupuli-test: rubocop.yml
```

Comment:
This is really optimized for our modules. I don't know if it would work well for puppet-lint gems.

Reply:
I was not sure neither.  I though that it can be used as a starting point, and if we do not need much change, maybe we can add them to the `rubocop.yml`; and if there are a lot of divergences, we can add a `puppet-lint-rubocop.yml` file.

So far, the only "real" issue is the one that follow these comments regarding the gemspec filename which makes rubocop unhappy.  Another issue is related to suppressing errors, and for this I would suggest adjusting the check in voxpupuli-test (currently it is completely forbidden (default), and we can adjust this to allow a comment to document why we want to ignore errors when loading optional simplecov).

I think it is good to have consistency across all code in the voxpupuli organization.